pkgs/top-level/python-packages.nix

```diff
@@ -935,7 +935,7 @@ in {
 
   proglog = callPackage ../development/python-modules/proglog { };
 
-  pure-python-adb-homeassistant = callPackage ../development/python-modules/pure-python-adb-homeassistant { };
+  pure-python-adb = callPackage ../development/python-modules/pure-python-adb { };
```

Comment:
this is a a breaking change, another option would be 
```suggestion
  pure-python-adb-homeassistant = callPackage ../development/python-modules/pure-python-adb { };
  
  pure-python-adb = self.pure-python-adb-homeassistant;
```

Reply:
I did it the other way around since pure-python-adb-homeassistant is the wrong package name